utils/src/main/java/org/robolectric/util/inject/Injector.java

```diff
@@ -238,8 +238,7 @@ public Injector build() {
     try {
       return ctor.newInstance(params);
     } catch (InstantiationException | IllegalAccessException | InvocationTargetException e) {
-      UnsafeAccess.throwException(e.getCause());
-      throw new IllegalStateException(); // unreachable
+      throw Util.sneakyThrow(e);
```

Comment:
Small behavior change here. I made the target method auto-unwrap `InvocationTargetException` since that's usually what you want. As a result, the other two exception types will not be unwrapped.